vaadin-text-field-flow-integration-tests/src/test/java/com/vaadin/flow/component/textfield/tests/ValueChangeModeIT.java

```diff
@@ -77,15 +112,15 @@ private void testValueChangeModes(WebElement field, String componentName) throws
                     "By default the value change events should be sent when clicking enter (ValueChangeMode should be ON_CHANGE)");
         }
 
-        field.sendKeys("a");
+        field.sendKeys("1");
```

Comment:
The number fields don't accept alphabets, so I changed this generic test method to work with each of the updated set of fields.